Dockerfile

```diff
@@ -93,6 +94,7 @@ RUN --mount=type=cache,target=/var/cache/apt --mount=type=cache,target=/var/lib/
         netbase \
         openssh-client \
         unzip \
+        wget \
```

Comment:
We already have `curl`, so please use that instead.

Reply:
Corrected.